regression/pages/__init__.py

```diff
@@ -0,0 +1,4 @@
+import os
+
+AUTH_USER_NAME = os.environ.get('BASIC_AUTH_USER', 'not_set')
```

Comment:
nit: could you call the global variables `BASIC_AUTH_USERNAME` AND `BASIC_AUTH_PASSWORD`?